### PR TITLE
Adopt Cookbook

### LINKED ISSUE #31: Sous-Chefs Cookbook setup
Cookbook needs setting up for sous

- [ ] metadata
- [x] terraform (admin)
- [x] standard files (bot)
- [x] cookstyle (bot)
- [x] labels standardise (bot)
- [ ] github actions
- [x] circleci (admin)
- [ ] rebase the files used. 
- [x] Remove old Webhooks

### PR DESCRIPTION
**Pull request by xorima**
# Description

Standardise everything

Signed-off-by: Jason Field <jason@avon-lea.co.uk>

## Issues Resolved

#31 

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This is a big Pull Request.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+name: ci
+
+on:
```
Comment (damacus) [inline .github/workflows/ci.yml]: Is this _still_ broken in that repo?

File: metadata.rb [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+supports 'debian'
+supports 'ubuntu'
 supports 'redhat'
+supports 'centos'
+supports 'fedora'
+supports 'amazon'
+supports 'scientific'
+supports 'oracle'
 
 depends 'compat_resource', '~> 12.14'
```
Comment (damacus) [inline metadata.rb]:
```suggestion
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (xorima) [reply .github/workflows/ci.yml]: ah, this is a manual copy and paste job, we can't automate this due to complexities. 

Why not do it the other way and have yamllint ignore this warning?

**Review by damacus — CHANGES_REQUESTED** [review]
As we're bumping the version we can get rid of this.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (damacus) [reply .github/workflows/ci.yml]: Mostly because it's an easy fix and on does mean something different in yaml. So it's a good habit to get into

**Comment by xorima** [comment]
And done

**Comment by xorima** [comment]
@damacus could you look at that one failing test on the rhel style boxes, it is `System Package nfs-utils` which works fine on a full fat vm, so it must be a dokken thing, however if I run dokken on my laptop is crashes :(

**Review by damacus — DISMISSED** [review]

**Comment by damacus** [comment]
I can do tomorrow. Knee deep in `bridge` :)

**Comment by nvwls** [comment]
Please allow chef-12.21 to still run

**Comment by xorima** [comment]
Plan to issue another PR after this to move around documentation and examples, but want to get something shipped.

File: README.md [hunk 3]
```diff
@@ -1,48 +1,44 @@
...
 This cookbook does not concern itself with managing an NFS server.
 
-Requirements
-------------
-- Chef 12.0.3 or higher
-- Ruby 1.9 or higher (preferably from the Chef full-stack installer)
-- Network accessible package repositories
+## Requirements
+
+- Chef 15 or higher
```
Comment (tas50) [inline README.md]:
```suggestion
- Chef Infra Client 12.21 or higher
```

**Comment by xorima** [comment]
v2.3.0 shipped

**Pull request merged**